include/avif/internal.h

```diff
@@ -473,6 +481,10 @@ avifBool avifROStreamReadU32(avifROStream * stream, uint32_t * v);
 avifBool avifROStreamReadU32Endianness(avifROStream * stream, uint32_t * v, avifBool littleEndian);
 avifBool avifROStreamReadUX8(avifROStream * stream, uint64_t * v, uint64_t factor); // Reads a factor*8 sized uint, saves in v
 avifBool avifROStreamReadU64(avifROStream * stream, uint64_t * v);
+#if defined(AVIF_ENABLE_EXPERIMENTAL_AVIR)
+avifBool avifROStreamReadBits(avifROStream * stream, uint32_t * v, size_t bitCount);
```

Comment:
It would be nice to have unit tests for these read/write functions.
Also I think they don't need to be hidden behind the flag.

Reply:
> It would be nice to have unit tests for these read/write functions.

I can add that to avifstreamtest.c once https://github.com/AOMediaCodec/libavif/pull/1504 is merged.

> Also I think they don't need to be hidden behind the flag.

Then it could be moved to another PR.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1506.